sdf.md

```diff
@@ -25,14 +25,20 @@ The JSON format of an SDF definition is described in this document.
   "defaultNamespace": "st",
   "odmObject": {
     "Switch": {
+      "id":0,
       "odmProperty": {
         "value": {
+        "id":1,
           "type": "string"
+          "enum": [
+            { "on":1 },
+            { "off":0 }
+          ]
         }
       },
       "odmAction": {
-        "on": {},
-        "off": {}
+        "on": {"id":3},
```

Comment:
Is this the revised format of enumerations?

Reply:
This is for "id" being a required quality of all definitions. There is another change in the examples for enum format.